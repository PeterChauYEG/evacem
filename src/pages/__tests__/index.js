import React from 'react'
import renderer from 'react-test-renderer'

// Component to be tested
import IndexPage from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(
    <IndexPage />
  )
  expect(tree).toMatchSnapshot()
})
