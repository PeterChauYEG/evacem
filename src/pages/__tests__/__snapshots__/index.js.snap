// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <h1>
      Edmonton Vietnamese
    </h1>
    <h1>
      Alliance Church
    </h1>
    <h2>
      English Ministries
    </h2>
  </div>
</div>
`;
